packages/rum-core/src/domain/rumEventsCollection/action/getSelectorsFromElement.ts

```diff
@@ -28,11 +28,26 @@ export function getSelectorsFromElement(element: Element, actionNameAttribute: s
       attributeSelectors
     )
   }
+  const globallyUniqueSelectorStrategies = attributeSelectors.concat(getIDSelector)
+  const uniqueAmongChildrenSelectorStrategies = attributeSelectors.concat(getClassSelector)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Maybe you can add `getTagNameSelector` to all `uniqueAmongChildrenSelectorStrategies` like so
```suggestion
  const uniqueAmongChildrenSelectorStrategies = attributeSelectors.concat(getClassSelector, getTagNameSelector)
```
And so simplify `getPositionSelector()` to return only `${element.tagName}:nth-of-type(${elementIndex!})`